### PR TITLE
Update build.yml

### LINKED ISSUE #151: Foreman installs the selene-light executable instead of selene on Linux
To reproduce, use [foreman](https://github.com/Roblox/foreman) to install the latest version of selene on Linux; attempting to use Roblox features such as `generate-roblox-std` will fail. Foreman selects the executable at the first index of a release archive when installing, so this might be happening because `selene-light` was zipped into `selene-0.8.0-linux.zip` before `selene`.

### PR DESCRIPTION
**Pull request by y3th**
Closes #151.

### CODE REVIEW

**Comment by Kampfkarren** [comment]
I'm confused, what exactly is this doing and how does this solve that issue? I think that issue is already solved even now that with Foreman I'm just dropping the zip files directly.

**Comment by y3th** [comment]
Foreman still ends up installing selene-light.

image

All this PR does is change the order in which the light version of selene and the full version of selene are built. Building the full version of selene first should solve the issue. 

Honestly, it'd be better if Foreman allowed you to choose an executable by name in an artifact.

**Comment by Kampfkarren** [comment]
I don't think it matters what the order is since it's just a zip file, @LPGhatguy can you bring any insight to this?

**Comment by LPGhatguy** [comment]
Selene's releases have more than one executable file in the same zip. Foreman just picks the first executable it finds, which looks like it might be the light one. I don't think this fix is very robust.

The light and full builds of Selene should probably be separate artifacts. Bundling both in the same zip is very strange to me, as you likely only want one or the other, and this blows up the size of the archive.

**Comment by Kampfkarren** [comment]
> Selene's releases have more than one executable file in the same zip. 

Correct.

I can see separating them into different zips. CC @y3th on if you want to do that.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -11,63 +11,78 @@ jobs:
...
-      run: |
-        cd selene
-        cargo build --locked --release --verbose --no-default-features
-        New-Item release -ItemType "directory"
-        Move-Item ..\target\release\selene.exe .\release\selene-light.exe
     - name: Build (All features)
       run: |
         cd selene
         cargo build --locked --release --all-features
+        New-Item release -ItemType "directory"
```
Comment (Kampfkarren) [inline .github/workflows/build.yml:18]: If we're going to be just shipping out the executables directly, I don't think we need to be making directories anymore--you can just send straight from target/release.

**Review by Kampfkarren — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (y3th) [reply .github/workflows/build.yml]: Done! I would've preferred to keep the build steps and the upload steps grouped, but this seems faster.

**Review by Kampfkarren — APPROVED** [review]

**Pull request merged**